test/factories/memberships.rb

```diff
@@ -16,6 +16,15 @@
       end
     end
 
+    trait :with_point_request do
+      after(:create) do |membership|
+        evaluation = create(:evaluation)
+        evaluation.update!(group: membership.group)
```

Comment:
What do you think passing the group to create instead of create & update?

```suggestion
        evaluation = create(:evaluation, group: membership.group)
```